docs/core/project-sdk/msbuild-props.md

```diff
@@ -1421,6 +1421,116 @@ The `RunWorkingDirectory` property defines the working directory for the applica
 </PropertyGroup>
 ```
 
+## Test project&ndash;related properties
+
+The following MSBuild properties are documented in this section:
+
+- [IsTestProject](#istestproject)
+- [IsTestingPlatformApplication](#istestingplatformapplication)
+- [Enable\[NugetPackageNameWithoutDots\]](#enablenugetpackagenamewithoutdots)
+- [EnableAspireTesting](#enableaspiretesting)
+- [EnablePlaywright](#enableplaywright)
+- [EnableMSTestRunner](#enablemstestrunner)
+- [EnableNUnitRunner](#enablenunitrunner)
+- [GenerateTestingPlatformEntryPoint](#generatetestingplatformentrypoint)
+- [TestingPlatformCaptureOutput](#testingplatformcaptureoutput)
+- [TestingPlatformCommandLineArguments](#testingplatformcommandlinearguments)
+- [TestingPlatformDotnetTestSupport](#testingplatformdotnettestsupport)
+- [TestingPlatformShowTestsFailure](#testingplatformshowtestsfailure)
+- [TestingExtensionsProfile](#testingextensionsprofile)
+- [UseVSTest](#usevstest)
+
```

Comment:
`GenerateTestingPlatformConfigurationFile` is only available when `IsTestingPlatformApplication` is `true` and is used to allow the copy and rename of the config file in the output folder.

Reply:
Tracked in https://github.com/dotnet/docs/issues/42544.